src/databases/ADIOS2/ADIOS2.xml

```diff
@@ -11,6 +11,7 @@
       avtADIOS2FileFormat.C
       avtMEUMMAPSFileFormat.C
       avtGTCFileFormat.C
+      avtLAMMPSFileFormat.C
```

Comment:
Is this related to zlib changes? If not, thats ok. Am not suggesting you make a new branch/commit/PR.